website/docs/reference/events-logging.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Events and logs"
+---
+
+:::info New in v1.0
+While dbt has always generated logs, the eventing + structured logging system is new in v1
+:::
+
+With every task that dbt performs, it generates events. It records those events as log messages, and writes them (in real time) to two places:
+- The command line console (`stdout`), to provide interactive feedback while running dbt
+- The debug log file (`logs/dbt.log`), to enable detailed [debugging of errors](debugging-errors) when they occur. The text-formatted log messages in this file include all `DEBUG`-level events, as well as contextual information, such as log level and thread name. The location of this file can be configured via [the `log_path` config](log-path).
+
+<File name='CLI'>
+
+```bash
+21:35:48  6 of 7 OK created view model dbt_testing.name_list......................... [CREATE VIEW in 0.17s]
+```
+
+</File>
+
+<File name='logs/dbt.log'>
+
+```text
+============================== 2021-12-02 21:29:35.417263 | c83a0afc-7ed3-49e7-8c0e-797af7f9d7b6 ==============================
+21:29:35.417263 [info ] [MainThread]: Running with dbt=1.0.0-rc3
+21:29:35.417955 [debug] [MainThread]: running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=None, defer=None, exclude=None, fail_fast=None, full_refresh=False, log_cache_events=False, log_format=None, partial_parse=None, printer_width=None, profile=None, profiles_dir='/Users/jerco/.dbt', project_dir=None, record_timing_info=None, rpc_method='run', select=None, selector_name=None, send_anonymous_usage_stats=None, single_threaded=False, state=None, static_parser=None, target=None, threads=None, use_colors=None, use_experimental_parser=None, vars='{}', version_check=None, warn_error=None, which='run', write_json=None)
+...
+21:29:35.814348 [debug] [Thread-1  ]: On model.my_project.my_table: BEGIN
+```
+
+</File>
+
+## Structured logging
+
+When `json` [log formatting](global-configs#log-formatting) is enabled, dbt will output produce rich, structured log information which can be piped into monitoring tools for analysis, or to power applications with dbt metadata in real time.
+
+Each log line will have the following JSON properties:
```

Comment:
Not a big deal for today's release, but this might be easier to read as a table and in alphabetical order. I know markdown tables are a PIA so maybe we can come up with a solution to that. 🤷🏻‍♀️

Reply:
I can do this!